core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -609,6 +609,8 @@ object VecInit extends SourceInfoDoc {
       // Bulk connecting two wires may not succeed because Chisel frontend does not infer
       // directions.
       (x, y) => x <> y
+    case _ =>
+      throw new Exception(s"Match Error: proto.direction: $proto.direction")
```

Comment:
This one looks like it is actually just a missing case that we need to support. Which direction is missing?

Reply:
Missing case added
```
    case ActualDirection.Empty =>
      (x, y) => x <> y
```